xbuild/src/gradle/ic_launcher.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+<!--
+  <background android:drawable="@mipmap/ic_launcher_background"/>
+-->
```

Comment:
is this comment useful, or should it be removed?

Reply:
If it's useful the user has to be reminded to extend `for variant in ["foreground", "monochrome"]`.

Reply:
It's a leftover from the [template](https://icon.kitchen/) I used. For most use cases I think people don't have to use an explicit background. In addition, the other platforms do not have a background for icon. So removing it may be the easiest.